src/client/cli/argparser.cpp

```diff
@@ -156,6 +156,18 @@ mp::ParseCode mp::ArgParser::commandParse(cmd::Command* command)
     if (!parsedOk)
     {
         cerr << qUtf8Printable(parser.errorText()) << '\n';
+
+        if (parser.errorText().contains("Unknown option"))
+        {
+            cerr << "\nYou need to separate the options to be passed to the command with \"--\", like this: \n";
```

Comment:
I think we could use a nicer message. Something like: "Options to the inner command should come after \"--\", like this:"